addons/account_analytic_default/models/account_analytic_default.py

```diff
@@ -75,6 +75,8 @@ class AccountMoveLine(models.Model):
     @api.depends('product_id', 'account_id', 'partner_id', 'date_maturity')
     def _compute_analytic_account(self):
         for record in self:
+            record.analytic_account_id = (record._origin or record).analytic_account_id
+            record.analytic_tag_ids = (record._origin or record).analytic_tag_ids
```

Comment:
last nitpick: this means that if an user set an analytic tag on the invoice line and then change partner the analytic tag will be overwritten, and usually we try to keep users modifications

Reply:
U turn: in this case it's ok, otherwise the computed value is useless
@nim-odoo r+?